base/linalg/tridiag.jl

```diff
@@ -14,6 +14,13 @@ immutable SymTridiagonal{T} <: AbstractMatrix{T}
     end
 end
 
+"""
+    SymTridiagonal(dv, ev)
+
+Construct a symmetric tridiagonal matrix from the diagonal and first diagonal,
+respectively. The result is of type `SymTridiagonal` and provides efficient specialized
+eigensolvers, but may be converted into a regular matrix with [`full`](:func:`full`). The length of `ev` must be one less than the length of `dv`.
```

Comment:
This is actually not true. It's one less or equal, e.g.
```jl
julia> SymTridiagonal(randn(5), randn(5))
5×5 SymTridiagonal{Float64}:
 -1.2657   -1.09259     ⋅           ⋅           ⋅      
 -1.09259  -0.094513   0.807025     ⋅           ⋅      
   ⋅        0.807025  -0.997063    0.0714734    ⋅      
   ⋅         ⋅         0.0714734  -0.716269    0.486895
   ⋅         ⋅          ⋅          0.486895   -1.403   
```
is okay. This allows for size zero `SymTridiagonal`s and some eigensolvers expect this as well.